app/cafe.py

```diff
@@ -0,0 +1,28 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(
+                f"{visitor["name"]} is not vaccinated.")
+
+        expiration_date = visitor["vaccine"].get("expiration_date")
+        if expiration_date < datetime.date.today():
+            raise OutdatedVaccineError(
+                "All friends should be vaccinated")
+
+        if not visitor.get("wearing_a_mask", False):
+            raise NotWearingMaskError(
+                f"{visitor["name"]} is not wearing a mask.")
```

Comment:
There is a syntax error in the string formatting here. You should use single quotes inside the f-string or escape the double quotes around `name`. For example: `f"{visitor['name']} is not wearing a mask."`